notebooks/geo/BepiColombo_FlybyVisualization.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    size = 4*12
</pre><p>You should have 4 in a constant NUM_COLUMNS and 12 NUM_LINES. That will make the code more explicit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>